src/main/scala/com/wavesplatform/account/Address.scala

```diff
@@ -30,11 +30,15 @@ object Address extends ScorexLogging {
 
   private class AddressImpl(val bytes: ByteStr) extends Address
 
+  private[wavesplatform] def createUnsafe(address: ByteStr): Address = {
```

Comment:
This should remain private. `fromBytes` preforms some additional validation and hence should be used wherever bytes need to be converted to an Address instance.